content/en/docs/k8s-operator/automatic.md

```diff
@@ -0,0 +1,170 @@
+[cert-manager](https://cert-manager.io/docs/installation/). If you use the helm
+chart, there is an option to generate a self-signed cert instead.
+
+## Create an OpenTelemetry Collector (Optional)
+
+It is a best practice to send telemetry from containers to an
+[OpenTelemetry Collector](../../collector/) instead of directly to a backend.
+The Collector helps simplify secret management, decouples data export problems
+(such as a need to do retries) from your apps, and lets you add additional data
+to your telemetry, such as with the `k8sattributesprocessor` component. If you
+chose not to use a Collector, you can skip to the next section.
+
+The Operator provides a
+[Custom Resource Definition (CRD) for the OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-operator/blob/main/docs/api.md#opentelemetrycollector)
+which is used to create an instance of the Collector that the Operator manages.
+The following example deploys the Collector as a deployment (the default), but
+there are other
+[deployment modes](https://github.com/open-telemetry/opentelemetry-operator#deployment-modes)
+that can be used.
+
+When using the `Deployment` mode the operator will also create a Service that
+can be used to interact with the Collector. The name of the service is the name
+of the `OpenTelemetryCollector` resource prepended to `-collector`. For our
+example that will be `demo-collector`.
+
+```bash
+kubectl apply -f - <<EOF
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: demo
+spec:
+  config: |
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+          http:
+    processors:
+      memory_limiter:
+        check_interval: 1s
+        limit_percentage: 75
+        spike_limit_percentage: 15
+      batch:
+        send_batch_size: 10000
+        timeout: 10s
+
+    exporters:
+      logging:
+
+    service:
+      pipelines:
+        traces:
+          receivers: [otlp]
+          processors: [memory_limiter]
+          exporters: [logging]
+        metrics:
+          receivers: [otlp]
+          processors: [memory_limiter]
+          exporters: [logging]
+EOF
+```
+
+The above command results in a deployment of the Collector that you can use as
+an endpoint for auto-instrumentation in your pods.
+
+## Configure Autoinstrumentation
+
+To be able to manage autoinstrumentation, the Operator needs to be configured to
+know what pods to instrument and which autoinstrumentation to use for those
+pods. This is done via the
+[Instrumentation CRD](https://github.com/open-telemetry/opentelemetry-operator/blob/main/docs/api.md#instrumentation).
+
+Creating the Instrumentation resource correctly is paramount to getting
+auto-instrumentation working. Making sure all endpoints and env vars are correct
+is required for auto-instrumentation to work properly.
+
+### .NET
+
+Coming Soon
+
+### Java
+
+Coming Soon
+
+### Node.js
+
+Coming Soon
+
+### Python
+
```

Comment:
Can you put a sentence to introduce what you are going to do with the following command?